website/blog/2023-09-28-how-to-contribute-to-open-source-2022.md

```diff
@@ -1,19 +1,19 @@
 ---
 slug: how-to-contribute-to-open-source-2022
-title: 'Hacktoberfest: How to be a Successful Open Source Contributor In 2022'
+title: 'Hacktoberfest: How to be a Successful Open Source Contributor'
 author: Elena Grahovac
-description: Contributing to open source might seem daunting, but you can start your tech journey during Hacktoberfest 2022.
+description: Contributing to open source might seem daunting, but you can start your tech journey during Hacktoberfest 2023.
 image: /img/blog/hacktoberfest.jpg
```

Comment:
That's the old logo

Reply:
It's probably better we update the image, and make it similar to other recent blog posts.

Reply:
I'm not sure why we insist on using the single file for two different images, but I'm sure it is not a good idea. File names are free. We _can_ have `hacktoberfest.jpg` and something like `hacktoberfest-2023.jpg` in the same directory

Reply:
On the same note – both the URL and file name mention 2022

Reply:
The URL is okay since we are republishing; we don't want to change the URL. Otherwise, the current link won't be reachable

Reply:
I don't understand why we are making it so needlessly complicated. Can we just create a new post instead of "republishing" with wrong logo and different years in the URL and text?

Reply:
That would mean creating a new blog post entirely from scratch without plagiarizing the previous one, even when 80% of the content will remain the same. This way, we are just updating the old blog posts with new information rather than rewriting the entire thing all over again. It's a normal process with old posts that just require a bit of new info.